tests/sentry/issues/endpoints/test_group_details.py

```diff
@@ -492,28 +440,6 @@ def test_resolved_in_next_release_no_release(self):
         response = self.client.put(url, data={"status": "resolvedInNextRelease"})
         assert response.status_code == 200, response.content
 
-        group = Group.objects.get(id=group.id, project=group.project.id)
-        assert group.status == GroupStatus.RESOLVED
-
-        # no GroupResolution because there is no release
-        assert not GroupResolution.objects.filter(group=group).exists()
-        assert response.data["statusDetails"] == {}
-
-    @with_feature("organizations:releases-resolve-next-release-semver-fix")
-    def test_resolved_in_next_release_with_flag_no_release(self):
```

Comment:
This test was to prevent normal releases from regressing with the new fix.

This is the test it duplicated:
https://github.com/getsentry/sentry/blob/1413cbf43a02eb0da8dad59ca641c3635c6e183d/tests/sentry/issues/endpoints/test_group_details.py#L429-L449